### ISSUE TITLE
Knwon issue? (sound starting after green flag)

### ISSUE THREAD

**Issue by sfederici** [leopard-js/leopard#189]
Is the fact that, even if the project starts immediately after translation, the sounds starts only after the green flag button has been pressed a known issue?

**Comment by adroitwhiz** [comment]
Yes it is, IIRC. The issue is that the Web Audio API only lets you start an `AudioContext` after a user interaction on the page has occurred. This is to prevent sites from auto-playing audio without any user interaction.

Does this issue only happen if starting the project immediately without any page interactions, or does it also occur after interacting with the page in some way but before the project viewer is appended to that same page? The former is unfixable due to the intentional design of the Web Audio API; the latter may be worth looking into.
